lib/msf/core/post/windows/process.rb

```diff
@@ -8,6 +8,7 @@ module Windows
 module Process
 
   include Msf::Post::Windows::ReflectiveDLLInjection
+  include Msf::Post::Process
```

Comment:
Do we need to add it to all of the system specifix mixins? i.e. these methods were also deleted from
lib/msf/core/post/linux/system.rb
lib/msf/core/post/solaris/system.rb
lib/msf/core/post/linux/system.rb

This looks like a breaking change to me, so we should probably include this line in the other places that were impacted - otherwise we can break modules that we don't own/maintain within this repository

I'm guessing that's why this module needed to explicitly add this mixin manually? 👀 
https://github.com/rapid7/metasploit-framework/pull/15260/files#diff-3f287c6a5c2c30af56a73b0479f3c378502dc75b8a3538e482b939bdae24e0da

Reply:
I guess one method was deleted in an earlier PR because it was not being used by any module.

Reply:
Hey, I looked at those 2 mixins and modules it's not using any methods from the `process` mixin, I guess this is now included in the modules which require it :)